packages/webcert/src/feature/certificate/Inputs/UeSickLeavePeriod/UeSickLeavePeriod.tsx

```diff
@@ -131,9 +140,30 @@ export const UeSickLeavePeriod: React.FC<Props> = ({ question, disabled }) => {
   }
 
   const handleWorkingHoursOnChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const inputNumber: number = +event.target.value.replace(/[^0-9]/g, '')
+
+    isWorkingHoursValid(inputNumber)
+
     setBaseWorkHours(event.target.value.replace(/[^0-9]/g, ''))
   }
 
+  const isWorkingHoursValid = (workingHours: number) => {
+    const error: ValidationError = {
+      category: question.parent,
+      id: question.id,
+      text: 'Ange ett giltigt antal arbetstimmar. Arbetstiden kan inte överstiga 168 timmar per vecka.',
+      type: 'WORKING_HOURS_ERROR',
+      field: 'WORKING_HOURS',
+      showAlways: true,
+    }
+
+    if (workingHours > 168) {
+      dispatch(updateClientValidationError({ shouldBeRemoved: false, validationError: error }))
+    } else {
+      dispatch(updateClientValidationError({ shouldBeRemoved: true, validationError: error }))
+    }
```

Comment:
You can simplify this code by doing the following
```suggestion
    const shouldBeRemoved = workingHours <= 168
    dispatch(updateClientValidationError({ shouldBeRemoved, validationError: error }))
```